<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convertor</class>
 <widget class="QMainWindow" name="convertor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конвертатор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="usd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>$</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usd_f">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="usd_s">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="usd_t">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="eur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>€</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eur_f">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eur_s">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eur_t">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>312</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
